tests/connection_expiry_test.go

```diff
@@ -23,13 +23,14 @@ func TestConnectionExpiry(t *testing.T) {
 	tenanted.SetConnMaxIdleTime(time.Second)
 	tenanted.SetConnMaxLifetime(time.Second)
 
-	// We use a random uuid here because we are just testing that PingContext is successfull
+	// We use a random uuid here because we are just testing that PingContext is successful
 	tc, ctx, err := tenancy.Open(ctx, tenanted.DB.DB, uuid.NewString())
```

Comment:
looks like we could set a fixed UUID here rather than using `uuid.NewString()` and it would drop the dependency on the `uuid` package?

Reply:
Yes that will work, good spot Chris